### PR TITLE
Industry 4.0 and Cybersecurity

### PR DESCRIPTION
**Pull request by collins-a**
Wrote article about Industry 4.0 and Cybersecurity
Included a .jpg as hero image

### CODE REVIEW

**Comment by hectorkambow** [comment]
Plagiarism check DONE

**Comment by collins-a** [comment]
Apologies for the Automated Fake News Detection commits. They are a mistake

File: articles/industry-4.0-and-cybersecurity/index.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+description: An introduction to Industry 4.0 through the enabling technologies, including fundamental principles of security, cybersecurity challenges, threats and insinuations.
+author: collins-agesa
+date: 2020-09-06T00:00:00-08:00
+topics: [Cybersecurity]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/industry-4.0-and-cybersecurity/hero.jpg
+    alt: bulma image example
+---
+Many technologies have emerged in recent times. These technologies include the Internet of Things, Big Data Analytics and Cyber-Physical Systems. They have enabled new and impactful initiatives such as Industry 4.0. this initiative is revolutionizing the manufacturing industry. Yet, the cybersecurity challenges associated with the technology have never been greater.
```
Comment (sudosoph) [inline articles/industry-4.0-and-cybersecurity/index.md:16]: The introduction is very generic and you introduce a lot of terms that are never defined. You also mention that Industry 4.0 revolutionizes manufacturing, yet don't specify how. I suggest giving an overview of industries that may be relevant, or rethinking your intro, and explaining why we should care about Industry 4.0 and cybersecurity in this section for context.

File: articles/industry-4.0-and-cybersecurity/index.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+<!--more-->
+
+## Industry 4.0
+
+Industry 4.0 had been a concept of German origin as a government initiative.  Dubbed as the fourth industrial revolution, it was first introduced as Industrie 4.0 in 2011. The general goal was to improve competitiveness of the manufacturing industry. It was also to advance the new generation of smart manufacturing systems. Enabling technologies played a big part to support this goal. These technologies include:
```
Comment (sudosoph) [inline articles/industry-4.0-and-cybersecurity/index.md:21]: Wording can be more concise. More specific details are needed about the fourth industrial revolution, as this is still very vague and the reader is left wondering what else they should know about.

File: articles/industry-4.0-and-cybersecurity/index.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+***Internet of Things (IoT):** a network of interconnected objects. Objects are physical devices and are billions in number globally, through the internet. IIoT refers to Industrial Internet of Things.
```
Comment (sudosoph) [inline articles/industry-4.0-and-cybersecurity/index.md:23]: Definition is incomplete. You define objects, which is unnecessary, but not what IOT does or consists of.

File: articles/industry-4.0-and-cybersecurity/index.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+***Cyber-Physical Systems:** integrations of physical processes with computation and networking processes. Key for automation and monitoring.
```
Comment (sudosoph) [inline articles/industry-4.0-and-cybersecurity/index.md:25]: This entire section on technologies features very basic definitions with little to no explanations of what they are/do and _why_ exactly they are relevant to Industry 4.0. Make sure to bring it back or relate it explicitly to your thesis.

File: articles/industry-4.0-and-cybersecurity/index.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+***Smart Factory:** advanced robotics, machine learning, additive manufacturing for rapid prototyping
+
+***Cloud:** (Big Data Analytics) for storage as well as computation among others.
+
+Depending on perspective, Industry 4.0 is a result of the Industrial Internet of Things. It is revolutionizing the manufacturing sector with innumerable benefits. Increase in efficiency, productivity, flexibility, agility are a few. Others include collaborative operation and sharing. The outcome includes lowering overhead, leading to increase profitability. Improved customer experience and facilitation of greater innovation and creativity is also noted.
```
Comment (sudosoph) [inline articles/industry-4.0-and-cybersecurity/index.md:31]: Here you are defining Industry 4.0 after briefly writing a line on Industrial Internet of Things earlier, as a quick aside to the IOT definition. 

You mention the innumerable benefits but the reader is puzzled and left wondering _how_.

File: articles/industry-4.0-and-cybersecurity/index.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+## Fundamental Principles of Security
+
+It is important to understand the Industry 4.0 cybersecurity landscape. We do this by first identifying the base principles of any security system. These outline the main objectives of information security.
```
Comment (sudosoph) [inline articles/industry-4.0-and-cybersecurity/index.md:35]: Vague intro sentence- can be scrapped or replaced. You jump right into the objectives of information security without discussing any of the vulnerabilities or flaws associated to Industry 4.0.

File: articles/industry-4.0-and-cybersecurity/index.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+
+### Confidentiality
+
+This is sustaining the secrecy of the flow of information. In this context, throughout the manufacturing system value chains.
```
Comment (sudosoph) [inline articles/industry-4.0-and-cybersecurity/index.md:39]: This doesn't mean anything.

File: articles/industry-4.0-and-cybersecurity/index.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+
+Industry 4.0 systems are complex yet flexible. This provides many points where attackers could intercept and tap information.
```
Comment (sudosoph) [inline articles/industry-4.0-and-cybersecurity/index.md:41]: How are they complex yet flexible? How do attackers intercept information? Where is this information even stored in the context of Industry 4.0?

File: articles/industry-4.0-and-cybersecurity/index.md [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+
+The damage caused by unauthorized access to information is costly. Imagine the interception of industrial industrial trade secrets. Confidentiality of the system must be upheld.
+
+### Integrity
+
+Integrity involves consistency, accuracy and reliability as well. This applies to the information coursing through the manufacturing system. There's an added focus on consistency, accuracy and reliability. This applies to the many physical components all through the supply chain.
+
+### Availability
+
+Physical attacks may render a manufacturing system out of service.
```
Comment (sudosoph) [inline articles/industry-4.0-and-cybersecurity/index.md:51]: What are physical attacks?

File: articles/industry-4.0-and-cybersecurity/index.md [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+
+It is important to understand the Industry 4.0 cybersecurity landscape. We do this by first identifying the base principles of any security system. These outline the main objectives of information security.
+
+### Confidentiality
+
+This is sustaining the secrecy of the flow of information. In this context, throughout the manufacturing system value chains.
+
+Industry 4.0 systems are complex yet flexible. This provides many points where attackers could intercept and tap information.
+
+The damage caused by unauthorized access to information is costly. Imagine the interception of industrial industrial trade secrets. Confidentiality of the system must be upheld.
```
Comment (sudosoph) [inline articles/industry-4.0-and-cybersecurity/index.md:43]: You could give specific examples of how it is costly.

File: articles/industry-4.0-and-cybersecurity/index.md [hunk 11]
```diff
@@ -0,0 +1,107 @@
...
+
+### Research efforts
+
+The integration of these emerging technologies has brought rapid changes to industries. The changes have brought about huge technological, social and economic impact. This means that we need more extensive research. Research can drive development of improved system architectures. It may help uphold transparency and trust through the value chain. This should be achieved without compromising the principles of security.
+
+## Final Note
+
+Industry 4.0 has the potential to be  he undisputed fourth industrial revolution. Yet, there is some way to go before it lives up to its full potential and application. Management or solving of the discussed challenges is key to the longevity of Industry 4.0. We should not underestimate the need for further research. It would go a long way to understand how to design  Industry 4.0 infrastructure. Research would also boost efforts to execute, secure and deploy effective infrastructure.
```
Comment (sudosoph) [inline articles/industry-4.0-and-cybersecurity/index.md:107]: This doesn't come out through your article. We don't feel that it is a revolution, as we are still struggling to understand what exactly it is and what makes it so revolutionary.

**Review by sudosoph — CHANGES_REQUESTED** [review]
This article requires a lot of work, or a re-write. There are no sources referenced and the writing is very vague. The reader is left wondering what Industry 4.0 is, how it affects manufacturing, what the links are with IOT and how the list of security issues that are prevalent across all technologies impede this sector specifically. Needs a lot more context on why the reader should care about Industry 4.0 and what makes it revolutionary. Every new term must be defined, and external sources must be referenced to provide supporting detail.

**Comment by hectorkambow** [comment]
@collins-a just pinging you to make sure you saw the Peer Reviewers comments and suggestions on your article 🚀

**Comment by collins-a** [comment]
Hey @hectorkambow @sudosoph. Thank you for the comments. Been working on the article. Hoping I'll have something much more fleshed out within the next few hours!

File: articles/industry-4.0-and-cybersecurity/index.md (hunk 11 above)
Comment (collins-a) [reply articles/industry-4.0-and-cybersecurity/index.md]: Hey @sudosoph! I have made a number of changes to the article. I hope I've addressed all the concerns. I'm aware it's quite a lengthy article now. I had so much content. Let me know if it's okay or if there are sections that can be trimmed, done away with, or split into two articles.

File: articles/industry-4.0-and-cybersecurity/index.md [hunk 12]
```diff
@@ -0,0 +1,210 @@
...
+
+## Final Note
+
+Industry 4.0 has the potential to be the undisputed fourth industrial revolution. Yet, there is some way to go before it lives up to its full potential and application. Management or solving of the discussed challenges is key to the longevity of Industry 4.0. We should not underestimate the need for further research. It would go a long way to understand how to design Industry 4.0 infrastructure. Research would also boost efforts to execute, secure and deploy effective infrastructure. Standardized policies and reworked cybersecurity frameworks that can accomodate Industry 4.0 systems from end to end need to be set in place for the success of this initiative.
+
+## References
+
+1.	Cybersecurity in the context of industry 4.0: A structured classification of critical assets and business impacts https://doi.org/10.1016/j.compind.2019.103165
```
Comment (hectorkambow) [inline articles/industry-4.0-and-cybersecurity/index.md:184]: please include references like links - see comment on #300

Comment (collins-a) [reply articles/industry-4.0-and-cybersecurity/index.md]: @hectorkambow done!

**Comment by hectorkambow** [comment]
@collins-a 
See message below - you may have to rebase your Forked repo - to clear this message
image

**Comment by hectorkambow** [comment]
@collins-a 
Note: Make sure your repository is up to date with the EngEd repository before contributing new articles.

only need to do this once
git remote add section http://github.com/section-io/engineering-education

then use this every time before contributing a new article
git fetch section

git pull --rebase section master

git push origin

Here is a link to a short video further explaining what a pull request is and how to fork a Github repository.

File: articles/industry-4.0-and-cybersecurity/index.md [hunk 13]
```diff
@@ -0,0 +1,208 @@
...
+
+
+### Industry 4.0 Enabling Technologies
+
+#### Internet of Things (IoT)
+
+The Internet of Things (IoT) refers to a network of interconnected objects. These objects are physical devices and are billions in number globally. From the term IoT, there are two parts; the internet and things. The connection is through the internet. Things are the billions of physical devices. This interconnection via the internet allows for cooperation and interaction in collecting, sharing and exchange of data. Due to cheap processors and wireless networks, it is easy to convert a physical object into part of the Internet of Things. Devices may be cars, sensors, buildings, machines and so forth. 
+
+The Internet of Things can be divided into Consumer IoT and Industrial IoT(IIoT). Consumer IoT defines smart connected product systems targeting an individual consumer. IIoT is centered around machine connectivity with an aim to not only to improve asset performance and product quality but also step up accountability and accountability throughout the systems.
```
Comment (hectorkambow) [inline articles/industry-4.0-and-cybersecurity/index.md:40]: "but also step up accountability and accountability throughout the systems." 

just wanted to clear up what the second accountability  - was meant to say?? @collins-a

Comment (collins-a) [reply articles/industry-4.0-and-cybersecurity/index.md:40]: Good eye @hectorkambow error of repetition on my part. Should be _step up accountability through the systems_

**Comment by collins-a** [comment]
@hectorkambow finally resolved the file conflict

**Comment by hectorkambow** [comment]
👋  Good morning/afternoon @jstangroome  - wanted to ask if you could give this "Industry 4.0 and Cybersecurity" article a final tech review when you have a moment. 
🚀  
Thank you

**Review by jstangroome — APPROVED** [review]
looks good to me

**Pull request merged**